drake/systems/plants/collision/ResultCollector.cpp

```diff
@@ -0,0 +1,19 @@
+#include <iostream>
+
+#include "drake/systems/plants/collision/DrakeCollision.h"
+#include "ResultCollector.h"
```

Comment:
I tried to split the difference and only call out code that could trivially be fixed _without_ solving all of the other (many!) defects in the DrakeCollision universe.

If "I'll fix that in a later PR" is a response to any of my comments, I will be content with that also.